sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -719,15 +724,17 @@ qualifiedName
     ;
 
 identifier
-    : strictIdentifier
+    : {ansi}? ansiReserved
+    | strictIdentifier
     | ANTI | FULL | INNER | LEFT | SEMI | RIGHT | NATURAL | JOIN | CROSS | ON
     | UNION | INTERSECT | EXCEPT | SETMINUS
```

Comment:
are these the only reserved keywords before your PR?

Reply:
Ah, I missed these keywords... I'm checking now.